src/app/containers/FastBtcDialog/components/TopUpHistory.tsx

```diff
@@ -38,7 +38,11 @@ export function TopUpHistory() {
   useEffect(() => {
     if (state.ready && account) {
       dispatch(actions.getHistory(account));
-
+      setHistory(
+        state.history.items.sort(
+          (x, y) => parseFloat(y.dateAdded) - parseFloat(x.dateAdded),
+        ),
+      );
```

Comment:
could this be moved to useMemo? 
it looks like effect hook will be reran each time `state` changes which may start loop of getHistory calls.

Reply:
Updated